test/regression/issue/08093.test.ts

```diff
@@ -0,0 +1,72 @@
+import { file, spawn } from "bun";
+import { afterAll, afterEach, beforeAll, beforeEach, expect, it } from "bun:test";
+import { bunExe, bunEnv as env } from "harness";
+import { access, writeFile } from "fs/promises";
+import { join } from "path";
+import {
+  dummyAfterAll,
+  dummyAfterEach,
+  dummyBeforeAll,
+  dummyBeforeEach,
+  dummyRegistry,
+  package_dir,
+  readdirSorted,
+  requested,
+  root_url,
+  setHandler,
+} from "./../../cli/install/dummy.registry.js";
+
+beforeAll(dummyBeforeAll);
+afterAll(dummyAfterAll);
+beforeEach(dummyBeforeEach);
+afterEach(dummyAfterEach);
+
+it("should install vendored node_modules with hardlink", async () => {
```

Comment:
Similar approach as tests in `test/cli/install/bun-install.test.ts` and more specifically test case `should prefer latest-tagged dependency`